config/params/network_config.go

```diff
@@ -58,3 +58,13 @@ func (c *NetworkConfig) Copy() *NetworkConfig {
 	}
 	return &config
 }
+
+// MaxRequestBlock determines the maximum number of blocks that can be requested in a single
+// request for a given epoch. If the epoch is at or beyond the mainnetDenebForkEpoch,
+// a special limit defined for Deneb is used.
+func MaxRequestBlock(e primitives.Epoch) uint64 {
+	if e >= mainnetDenebForkEpoch {
```

Comment:
wouldn't this value change based on the specific network? like interop or e2e or minimal?

Reply:
[b1c3032](https://github.com/prysmaticlabs/prysm/pull/13173/commits/b1c30322bb22cfd08a7695d26040748f709fd2b1)

Reply:
nice catch @james-prysm